src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -50,15 +51,17 @@ public class QueueStatsService {
     private final DequeueStatisticCollector dequeueStatisticCollector;
     private final RedisQuesExceptionFactory exceptionFactory;
     private final Semaphore incomingRequestQuota;
+    private final RedisquesConfiguration modConfig;
 
     public QueueStatsService(
-        Vertx vertx,
-        EventBus eventBus,
-        String redisquesAddress,
-        QueueStatisticsCollector queueStatisticsCollector,
-        DequeueStatisticCollector dequeueStatisticCollector,
-        RedisQuesExceptionFactory exceptionFactory,
-        Semaphore incomingRequestQuota
+            Vertx vertx,
+            EventBus eventBus,
+            String redisquesAddress,
+            QueueStatisticsCollector queueStatisticsCollector,
+            DequeueStatisticCollector dequeueStatisticCollector,
+            RedisQuesExceptionFactory exceptionFactory,
+            Semaphore incomingRequestQuota,
+            RedisquesConfiguration modConfig
```

Comment:
I would suggest to only provide a boolean value like `fetchQueueStats` instead of the whole `RedisquesConfiguration` object

Reply:
ok I will implement.